core/src/test/scala/com/rallyhealth/vapors/core/interpreter/FoldOutputSpec.scala

```diff
@@ -0,0 +1,29 @@
+package com.rallyhealth.vapors.core.interpreter
+
+import com.rallyhealth.vapors.core.data.FactTable
+import com.rallyhealth.vapors.core.dsl._
+import org.scalatest.freespec.AnyFreeSpec
+
+class FoldOutputSpec extends AnyFreeSpec {
+
+  "Expr.FoldOutput" - {
+
+    "fold a list of ints into its sum" in {
+      val query = const(List(1, 2, 3)).withOutputFoldable.fold
+      val result = eval(FactTable.empty)(query)
+      assertResult(6)(result.output.value)
```

Comment:
Is it worthwhile to have a different Monoid supplied to change the behaviour?

Reply:
I suppose this is just baked into cats now, so yea, you would have to supply your own monoid for `Int` if you wanted to do something other than addition. Maybe I could define a specific `.sum` and `.product` that are strict about the type of monoid they use?

Reply:
Either way, this PR meets our needs for now, so I am going to move forward and we can revisit this before v1